dev/build-docs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# build the base image
+docker build -t databricks/graphframes --build-arg PYTHON_VERSION=2.7 .
```

Comment:
Do we need Python 2.7?

Reply:
I used python2.7 because this is in the sparkdl docs/Dockerfile

# An docker image `databricks/sparkdl` need to be generated by run
#     $docker build -t databricks/sparkdl --build-arg PYTHON_VERSION=2.7 .
# on base directory. Currently we need to build it with Python2.

Reply:
Yes, because we cannot build with Python 3 there. If we have to use Python 2 here, it would be nice to add an inline comment to explain why we don't use the default one.